subprojects/docs/src/docs/release/notes.md

```diff
@@ -84,22 +169,34 @@ val doesntMatch = property.filter { it.contains("baz") } // is empty
 
 Note that `filter` is evaluated lazily, so the predicate is only evaluated when the value is queried.
 
-<!-- ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-ADD RELEASE FEATURES ABOVE
-==========================================================
-
--->
-
 ## Promoted features
 
 Promoted features are features that were incubating in previous versions of Gradle but are now supported and subject to backwards compatibility.
 See the User Manual section on the “[Feature Lifecycle](userguide/feature_lifecycle.html)” for more information.
 
 The following are the features that have been promoted in this Gradle release.
 
-<!--
-### Example promoted
--->
+<a name="assign-stable"></a>
+### Simple property assignment in Kotlin DSL is now stable
+
+[Gradle 8.1](/8.1.1/release-notes.html#experimental-simple-property-assignment-in-kotlin-dsl-scripts) introduced a simpler way to assign values to lazy property types in Kotlin scripts using the `=` operator instead of the `set()` method.
+This new assignment operator was enabled by default in 8.2 and in this release it is marked stable.
+
+```kotlin
+interface Extension {
+    val description: Property<String>
+}
+
+// register "extension" with type Extension
+extension {
+    // Using the set() method call
+    description.set("Hello Property")
+    // Incubating: using lazy property assignment
```

Comment:
❓ Why is this incubating - my understanding is that this is now stable, right?

Reply:
You are right Stefan, thank you. I'll fix it.